contrib/core/actions/send_mail/send_mail

```diff
@@ -83,7 +83,7 @@ EOF
 
   cat <<EOF
 --${boundary}
-Content-Type: ${CONTENT_TYPE}
+Content-Type: ${CONTENT_TYPE}; charset=UTF-8
```

Comment:
I didn't add it here because I thought we don't need it here. This section refers to attachments which should alrady be base64 encoded so it should only contain ascii characters.

Having said that, if it works fine and has no negative side affects, I'm fine with keeping it here.

Reply:
I agree that it is a bit odd, but when I did not include this, I got garbled non-ascii characters in the e-mail. Not sure why though.

Reply:
Edit: I was wrong, this part also refers to the html version of the body. The section above refers to the attachments so it's correct to have charset here as well.